org.muttum.muttum.yml

```diff
@@ -0,0 +1,196 @@
+        /* Not all linuxes have yacc, but all have bison */
+        #define YACC "bison -y"
+
+        #define TERMLIB       "-ltinfo"
+
+        #define BINDIR	"/app/bin"
+        #define LIBDIR	"/app/lib"
+        #define MAN1DIR	"/app/man/man1"
+        #define	MAN45DIR "/app/man/man5"
+        #define	MAN45EXT ".5"
+
+        #endif /* LOCAL_H_INCLUDED */
+        EOF
+      - make all
+      - make install
+    sources:
+      - type: archive
+        url: https://www.cs.hmc.edu/~geoff/tars/ispell-3.4.06.tar.gz
+        sha256: 17c91633d4c8075acc503163a16463fc54ab1c7453280ad39cd3db75c783eba6
+        strip-components: 1
+  - name: english-words
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/org.muttum.muttum
+      - >-
+        cat final/english-words.* final/american-words.*
+        | sort -u
+        | iconv -f iso-8859-1 -t utf-8 -o /app/share/org.muttum.muttum/en_US
+      - >-
+        cat final/english-words.* final/australian-words.*
+        | sort -u
+        | iconv -f iso-8859-1 -t utf-8 -o /app/share/org.muttum.muttum/en_AU
+      - >-
+        cat final/english-words.* final/british-words.*
+        | sort -u
+        | iconv -f iso-8859-1 -t utf-8 -o /app/share/org.muttum.muttum/en_GB
+      - >-
+        cat final/english-words.* final/canadian-words.*
+        | sort -u
+        | iconv -f iso-8859-1 -t utf-8 -o /app/share/org.muttum.muttum/en_CA
+      - cp Copyright /app/share/org.muttum.muttum/en.license
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/wordlist/scowl-2020.12.07.tar.gz
+        sha256: 5587667caa20c4891390c2d42dbb4d5c4c3f41bee77af1457ece3ba23fb859cc
+        strip-components: 1
+  - name: german-words
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/org.muttum.muttum
+      - make isowordlist-de_AT
+      - >-
+        iconv -f iso-8859-1 -t utf-8 < isowordlist-de_AT
+        > /app/share/org.muttum.muttum/de_AT
+      - make isowordlist-de_CH
+      - >-
+        iconv -f iso-8859-1 -t utf-8 < isowordlist-de_CH
+        > /app/share/org.muttum.muttum/de_CH
+      - make isowordlist-de_DE
+      - >-
+        iconv -f iso-8859-1 -t utf-8 < isowordlist-de_DE
+        > /app/share/org.muttum.muttum/de_DE
+      - cp Documentation/Copyright /app/share/org.muttum.muttum/de.license
+    sources:
+      - type: archive
+        url: https://j3e.de/ispell/igerman98/dict/igerman98-20161207.tar.bz2
+        sha256: 17296f03c5fea62d76ecc530ebe80f6adc430278f58d472dc1842d71612960a8
+        strip-components: 1
+  - name: french-words
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/org.muttum.muttum
+      - cp french /app/share/org.muttum.muttum/fr
+      - cp LICENSES/GPL-3.0-or-later.txt /app/share/org.muttum.muttum/fr.license
+    sources:
+      - type: git
+        url: https://salsa.debian.org/gpernot/wfrench.git
+        commit: d69373cc8e4b0f0629378856c37a5df6f8c3b155
+  - name: muttum.toml
```

Comment:
I don't see why this is not part of the main module. And why its build system can't generate it.

Reply:
The main module only builds the game engine and not any data are provided by it, so every distributor can use their already existing dictionnaries and link muttum with them with the `muttum.toml` configuration file.

The advantage of the current manifest file is that it documents exactly dictionaries sources and how to build them.

If Flathub prefer, I can build the dictionaries myself and copy them aside the manifest file (like I asked for the `config.toml` file above).

Reply:
you probably misunderstand what I meant by "module". `muttum.toml` is a module. `muttum` is another module.

Reply:
still no done.

Reply:
I've just merged the `french-words` and `muttum.toml` modules into a new `external-files` module.

Would you prefer I merge them into the main `muttum` module ?